delocate/tools/win32.py

```diff
@@ -0,0 +1,183 @@
+import os
+import zipfile
+
+from bindepend import getImports
+from machomachomangler.pe import redll
+from .common import InstallNameError
+
+
+def get_install_names(filename):
+    """ Return install names from library named in `filename`
```

Comment:
Is 'install names' a thing in Windows?   Would it be better called 'DLL names'?   If not maybe an explanation in the docstring?